src/resources/views/signin.ejs

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+  <title>DSC-DUT | Shorten URL tool</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Google+Sans&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="/css/style.css" />
+  <link rel="stylesheet" href="/css/loader.min.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="shortcut icon" href="/img/favicon.ico" type="image/x-icon" />
+</head>
+
+<body>
+  <div id="app">
+    <div id="app-logo">
+      <img src="/img/logo.png" alt="DSC-DUT Logo" />
+    </div>
+    <a id="app-button-shorten" class="app-button shorten" type="submit" href="./">Shorten URL</a>
+    <div id="app-container">
+      <h1>Sign in to manage your URLs</h1>
+      <form id="app-form">
+        <input type="email" class="app-input" id="app-input-email" placeholder="Input your email" />
+        <div class="app-input-container">
+          <input type="password" id="app-input-password" class="app-input password" placeholder="Input your password" />
+          <i id="eye-password" class="eye-password eye-signin far fa-eye-slash"></i>
+        </div>
+        <div class="app-remember-forgot">
+          <div class="app-remember-container">
+            <input type="checkbox" class="checkbox-remember"/>
+            <span>Remember me</span>
+          </div>
+          <a href="./forgot.ejs" class="forgot-password">Forgot your password?</a>
+        </div>
```

Comment:
Is the `.ejs` extension required?

Reply:
In Ejs, the *href* attribute should only contain the route, such as  */forgot*. The *.* is redundant and the *.ejs* extension may not work.